.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
                 -p 8001:8080 \
                 --name nominatim \
                 nominatim &
-              sleep 120
+
```

Comment:
I've tried a few times to get rid of these sleeps but always failed. I'm going to run the tests a few times to see if there is any difference now.

Reply:
According to the python script, it should automatically retry failed connection with an exponential backoff.
Hard-coding a forced sleep may prolong the test more than necessary, while the container may be already ready before the timeout. IMHO a polling HTTP call done periodically by the script sounds more snappy than the forced `sleep`.

Reply:
It still wasn't enough though. I will run it a few times and see.